samples/resources/README.md

```diff
@@ -0,0 +1,87 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+description: "These samples will show you how to manage reosurce in resource sdk."
+urlFragment: azure-samples-python-management
+---
+
+# Python Management SDK Samples
+
+These samples will show you how to manage reosurces in resource sdk.
+
+## Features
+
+This project framework provides examples for the following services:
+
+### Resource
+* Using the **Management Resource SDK** [azure-mgmt-resource](https://pypi.org/project/azure-mgmt-resource/) for the [Azure Resource Management API](https://docs.microsoft.com/en-us/rest/api/resources/)
+
+## Getting Started
+
+### Prerequisites
+
+1.  Before we start these samples, we need [register a new application using Azure portal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal). Then copy the `Subscription ID`, `Application (client) ID`, `Directory (tenant) ID` and create a new application secret to get the secret value.
+
+2. Put your secret info into environment. The example in linux bash:
```

Comment:
Store your credentials an environment variables.
For example, in Linux-based OS, you can do